streams/src/main/java/org/apache/kafka/streams/processor/internals/StandbyTask.java

```diff
@@ -92,7 +92,9 @@ public void resume() {
     public void commit() {
         log.trace("{} Committing", logPrefix);
         stateMgr.flush();
-        stateMgr.checkpoint(Collections.<TopicPartition, Long>emptyMap());
+        if (!eosEnabled) {
```

Comment:
While reading the code: when we call `StandbyTask.close()` it calls `commit()` but not `suspend()`; with this change when EOS is turned on we will not write checkpoint files when closing the standby task, I think this is not right?